frontend/www/js/omegaup/arena/contestStore.ts

```diff
@@ -0,0 +1,50 @@
+import Vue from 'vue';
+import Vuex, { Commit } from 'vuex';
+import * as api from '../api';
+import { messages, types } from '../api_types';
+
+Vue.use(Vuex);
+
+export interface ContestState {
+  // The map of contest lists.
+  contests: types.TimeTypeContests;
+}
+
+export interface NamedContestListRequest {
+  name: string;
+  requestParams: messages.ContestListRequest;
+}
+
+interface NamedContestListResponse {
+  name: string;
+  response: messages.ContestListResponse;
+}
+
+export const contestStoreConfig = {
+  state: {
+    contests: {},
+  },
+  mutations: {
+    updateAll(state: ContestState, payloadContests: types.TimeTypeContests) {
+      state.contests = { ...state.contests, ...payloadContests };
+    },
+    updateList(
+      state: ContestState,
+      { name, response }: NamedContestListResponse,
+    ) {
+      Vue.set(state.contests, name, response);
+    },
+  },
+  actions: {
+    getContestList(
```

Comment:
```suggestion
    fetchContestList(
```

para que no se confunda con un getter.

Reply:
gracias!